components/ChoreWeight.tsx

```diff
@@ -48,15 +57,19 @@ export default function ChoreWeight({
           <Card.Actions>
             <View style={s.content}>
               <View style={s.textContainer}>
-                <Text style={{ fontWeight: 'bold', fontSize: 18 }}>
+                <Text
+                  style={{ fontWeight: 'bold', fontSize: 18, color: textColor }}
+                >
```

Comment:
Color is not needed here as the theme takes care of this
```suggestion
                <Text
                  style={{ fontWeight: 'bold', fontSize: 18 }}
                >
```